SergiyGoryainov/HW03/HW03-2.py

```diff
@@ -0,0 +1,15 @@
+four_digit_int = list(input('Enter 4-digit whole number '))
```

Comment:
good job
look at this implementation:
value_int = int(input('Enter 4-digit number:'))

v_int = value_int

multi = 1

while True:
    multi *= v_int % 10
    v_int = v_int // 10
    if v_int // 10 == 0:
        multi *= v_int
        break
print(f"Multiplication of digits = {multi}")
print(f"Revers of number = {str(value_int)[::-1]}")
print(f"Sorted of number = {''.join(sorted(str(value_int)))}")